docs/deploy/index.md

```diff
@@ -109,7 +109,9 @@ You can see that IP address or FQDN with the following command:
 kubectl get service ingress-nginx-controller --namespace=ingress-nginx
 ```
 
-Set up a DNS record pointing to that IP address or FQDN; then create an ingress resource. The following example assumes that you have set up a DNS record for `www.demo.io`:
+It will be the `EXTERNAL-IP` field. If that field shows `<pending>`, this means that your Kubernetes cluster wasn't able to provision the load balancer (generally, this is because it doesn't support services of type `LoadBalancer`).
+
+Once you have the external IP address (or FQDN), set up a DNS record pointing to it. Then you can create an ingress resource. The following example assumes that you have set up a DNS record for `www.demo.io`:
```

Comment:
Wondering if it makes sense to mention /etc/hosts entries and curl flags for name to ipaddress resolution.

Reply:
I try to avoid recommending editing /etc/hosts, because it doesn't work on Windows, and I've seen lots of people forget that they've set up a manual entry there and wonder why things don't work later 😅

I'm not sure what you mean about the curl flags though!